### DISCUSSION TITLE
A Custom Icon

### DISCUSSION THREAD

**Discussion by davidbosman** [kmwallio/ThiefMD#100] [General]
I quite like the 'Thief' idea, but having a bad eyesight I find it hard to see the small thief in the icon. So I took the liberty to make my own... simplified version, better suited to my needs, to use on a custom launcher (I have no idea how to change the app default icon).

I don't know if it is something you would consider updating sometime in a future release but here is a [screenshot](https://davidbosman.fr/tux/images/Thief.jpg). The icon is SVG, so if you would ever want to use it, just let me know ;)

**Comment by kmwallio** [comment]
Thank you for sharing! I like how it has like a fountain pen for a goatee.

If/when we redesign the icon, we're aiming to follow [GNOME's Icon Recommendations](https://developer.gnome.org/hig/stable/icon-design.html.en). I might steal the concept/idea though :stuck_out_tongue_closed_eyes:. A book/page/computer that shows the writing intent that also makes up the cat burglar is cool.

**Reply by davidbosman** [reply]
I didn't know about these recommandations. That's great, I will read it thoroughly and maybe try to do a new version of the icon (btw, I like the goatee too) :wink:
